### PR TITLE
Letter Case Permutation using recursion

### PR DESCRIPTION
**Pull request by 1711shashank**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Letter Case Permutation using recursion

### Checklist

- [✔] I've read the contribution guidelines.
- [✔] I've checked the issue list before deciding what to submit.
- [✔] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1645

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@1711shashank Please remove the unnecessary file and commits.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@1711shashank I dont think `All_possible_subset_using_recursion.cpp` and `Print_N_bit_binary_numbers_having_more_1s_than_0s_using_recursion.cpp` programs belongs to this PR.

File: C-Plus-Plus/recursion/Letter_Case_Permutation_using_recursion.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        str.erase(str.begin() + 0);
+
+        solve(output3, str);
+    }
+}
+
+int main()
+{
+    string str;
+    cout << "Enter an alphanumeric string : "
```
Comment (iamrajiv) [inline C-Plus-Plus/recursion/Letter_Case_Permutation_using_recursion.cpp]: @1711shashank Expected `;` here.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@1711shashank `op1`, `op2`, `op3` has not been declared.

File: C-Plus-Plus/recursion/Letter_Case_Permutation_using_recursion.cpp (hunk 1 above)
Comment (1711shashank) [reply C-Plus-Plus/recursion/Letter_Case_Permutation_using_recursion.cpp]: Sorry, 
By Mistake i have added the different/untested files

**Comment by 1711shashank** [comment]
> @1711shashank I dont think `All_possible_subset_using_recursion.cpp` and `Print_N_bit_binary_numbers_having_more_1s_than_0s_using_recursion.cpp` programs belongs to this PR.

Can you tell me how to create new pull request from same branch when previous pull request is not merged.

Because whenever I am trying to add another file for some another issue it automatically added/linked to the previous unmerged PR.
And I am not getting the option for **create new PR**

I also asked this question in the slack but didn't get any response

**Comment by shraddhavp** [comment]
http://ndlib.github.io/practices/one-commit-per-pull-request/ 

you can follow this @1711shashank

**Comment by 1711shashank** [comment]
@shraddhavp Thank You 👍

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Comment by 1711shashank** [comment]
@HarshCasper I have already done the required changes, can you mention what more changes are required.
I think there are no activity in this PR because I am waiting to the Review.

**Pull request closed without merging**